### DISCUSSION TITLE
Tabs的子控件props第一次变化不能被watch监听

### DISCUSSION THREAD

**Discussion by holyrong** [vueComponent/ant-design-vue#4432] [Q&A]
**ant-design-vue 2.2.2**

父组件：
 
![image](https://user-images.githubusercontent.com/8318475/127110059-37fa04f6-4330-4c07-9e35-1fc075e61b9c.png)

![image](https://user-images.githubusercontent.com/8318475/127109927-36bdcc53-b02b-4a97-b91f-e1e1e53eae28.png)

 
子组件：
 
![image](https://user-images.githubusercontent.com/8318475/127109938-cd54d1ba-ae15-4f07-9a1d-e719f3146888.png)
![image](https://user-images.githubusercontent.com/8318475/127109952-1524550c-5a1d-45f4-ad61-da0b404093d4.png)

 
运行效果：
 
![image](https://user-images.githubusercontent.com/8318475/127109963-81f4c78c-57e9-4dcc-9932-748d362488f4.png)

为什么在第一次切换tab到user时，staff-user子组件的watch监听不到变化？如果子组件不放到tabs中，可以正常监听props的变化。

**Comment by tangjinzhou** [comment]
加 immediate

因为 tabs 是懒加载的，没有激活之前是不渲染到
